BlueprintUI/Tests/BlueprintViewTests.swift

```diff
@@ -2,6 +2,70 @@ import XCTest
 @testable import BlueprintUI
 
 class BlueprintViewTests: XCTestCase {
+    
+    func test_sizeThatFits() {
+        
+        let blueprintView = BlueprintView()
+        
+        // Normal sizes.
+        
+        do {
+            let element = MeasurableElement { constraint in
+                XCTAssertEqual(constraint.maximum, CGSize(width: 200, height: 100))
+                return CGSize(width: 100, height: 50)
+            }
+            
+            blueprintView.element = element
+            
+            XCTAssertEqual(blueprintView.sizeThatFits(CGSize(width: 200, height: 100)), CGSize(width: 100, height: 50))
+        }
+        
+        // Unconstrained in both axes.
+        
+        do {
+            let element = MeasurableElement { constraint in
+                XCTAssertEqual(constraint.maximum, CGSize(width: CGFloat.greatestFiniteMagnitude, height: CGFloat.greatestFiniteMagnitude))
```

Comment:
We should assert that the constraint is explicitly `unconstrained`. The existing behavior would have been the same for `maximum`.

Reply:
Ooh yeah, good point

Reply:
Done!